olm-catalog/serverless-operator/operator.environment.yaml

```diff
@@ -0,0 +1,24 @@
+---
+openshift:
+  knative:
+    serving:
+      branch: release-v0.13.3
+      ci-promotion-name: knative-v0.13.3
+    eventing:
+      branch: release-v0.13.0
+      ci-promotion-name: knative-v0.13.0
+    eventing-contrib:
+      branch: release-v0.15.0
+      ci-promotion-name: knative-v0.15.0
+    kafka:
+      branch: v0.15.0
```

Comment:
kafka/camel/pipelines settings aren't image-specific, they refer to current releases of their respective operators, so I don't think they should be maintained here...

Reply:
It's not image specific parameters. Those specify serverless supported environment. So, in relation to kafka/camel/pipelines it describe what version we test that with.

Reply:
the kafka.branch and camel.branch parameters aren't actually used by jobs by default... (by default, the current versions of knative camel and kafka operators from the operatorhub are installed) .. the .branch parameters are only used when installing unreleased versions)

Reply:
we should probably just change that to just specify channels, as in the automatic triggers we will probably only want to test with released versions of those...

Reply:
I think not necessarily. The goal is to be able to trigger all jobs that would test the release, against all we are supporting.